src/pages/studyView/charts/ChartContainer.tsx

```diff
@@ -455,6 +491,55 @@ export class ChartContainer extends React.Component<IChartContainerProps, {}> {
         return false;
     }
 
+    getMutationPlotControls() {
```

Comment:
Let's define a separate component in a separate file, like `StudyViewMutationPlotControls`, if possible to keep ChartContainer cleaner